pyxdf/pyxdf.py

```diff
@@ -300,7 +307,11 @@ def __init__(self, xml):
                             temp[StreamId].last_timestamp = stamps[k]
                             # read the values
                             raw = f.read(temp[StreamId].samplebytes)
-                            values[k, :] = struct.unpack(temp[StreamId].structfmt, raw)
+                            # no fromfile(), see
+                            # https://github.com/numpy/numpy/issues/13319
+                            values[k, :] = np.frombuffer(raw,
```

Comment:
frombuffer doesn't need a character format string and is about 30% faster in one of my tests